Maestro/subscriptions.json

```diff
@@ -142,54 +90,43 @@
             "/p:NotifyGitHubUsers=dotnet/coreclr-contrib",
             "/verbosity:Normal"
           ]
-        },
-        // This handler will bring the Latest CoreFX master build into WCF master
-        {
-          "maestroAction": "wcf-general",
-          "maestroDelay": "00:10:00",
-          "ScriptFileName": "build-managed.cmd",
-          "Arguments": [
-            "--",
-            "/t:UpdateDependenciesAndSubmitPullRequest",
-            "/p:GitHubUser=dotnet-bot",
-            "/p:GitHubEmail=dotnet-bot@microsoft.com",
-            "/p:GitHubAuthToken=`$(`$Secrets['DotNetBotGitHubPassword'])",
-            "/p:ProjectRepoOwner=dotnet",
-            "/p:ProjectRepoName=wcf",
-            "/p:ProjectRepoBranch=master",
-            "/p:NotifyGitHubUsers=dotnet/wcf-contrib",
-            "/verbosity:Normal"
-          ]
         }
-      ]
+      }
     },
+    // Update dependencies in CoreCLR release/1.0.0
```

Comment:
I was referring to the property name, to make this similar to the other 1.0.0 subscriptions like `// Update CoreCLR dependencies in CoreFX release/1.0.0`.

It could also be `Update ProjectK TFS dependencies in CoreCLR release/1.0.0`.